src/PhotometryMapping.cc

```diff
@@ -23,30 +23,49 @@ void ChipVisitPhotometryMapping::computeParameterDerivatives(MeasuredStar const
 
     // NOTE: the chip derivatives start at 0, and the visit derivatives start at the last chip derivative.
     // This is independent of the full-fit indices.
-    Eigen::Ref<Eigen::VectorXd> chipBlock = derivatives.segment(0, _chipMapping->getNpar());
```

Comment:
Yes, although the code that it applies to now lives inside the two if statements, instead of on the immediately-following lines. Suggestions on how to make that clearer?

Reply:
What's wrong with

     // NOTE: the chip derivatives start at 0. This is independent of the full-fit indices.
    ...
     // NOTE: the visit derivatives start at the last chip derivative. This is independent of the full-fit indices.